### PR TITLE
fix(components): Remove default value for prop size in ConfirmationModal

### PR DESCRIPTION
**Pull request by igor-m-jobber**
## Motivations
In the [PR](https://github.com/GetJobber/atlantis/pull/969) that added additional functionality to the `ConfirmationModal` we assign the default `size` property to the `small` value. This led to only 2 sizes being available for the modal size: `small` and `large`. 

We want to ensure that if `size` property is not supplied, none of the styles for the size modal will be applied, meaning that it will be the default size.

## Changes
Remove the default value for the `size` prop in the `ConfirmationModal` component.

### Added
n/a

### Changed
The `small` size prop was a default value. Now, `size` doesn't have default value.

### Deprecated
n/a
### Removed
n/a
- 

### Fixed
n/a
- 

### Security
n/a
- 

## Testing
Go to the Confirmation modal and open it. Should open the modal.
If you provide `small` or `large` size value - the size of the modal should change correspondingly.  

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Review by MichaelParadis — APPROVED** [review]
Code looks good. Going to wait on @chris-at-jobber 's input before merged though

**Comment by chris-at-jobber** [comment]
@MichaelParadis this looks alright, but this seems like a sign that we should add an explicit `base` size property to `Modal`. The issue here seems less that the `ConfirmationModal` defaults to `small` than that there's no way to use the standard modal size if desired.

**Comment by MichaelParadis** [comment]
@chris-at-jobber it looks like the reason for Modal not having a base size is by default it appears to default to 100% width to fit the content while small and large limits the size. I could be wrong though

https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Modal/Sizes.css#L1

https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Modal/Modal.css#L33-L45

**Comment by chris-at-jobber** [comment]
Modal has a max-size at default: 

Defined [here](https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Modal/Modal.css#L39) as `max-width: --modal-width`  

And the base value is set [here](https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Modal/Modal.css#L2) as `--modal--width: calc(var(--base-unit) * 37.5);`

**Comment by igor-m-jobber** [comment]
Hey @MichaelParadis @chris-at-jobber. 
Wondering if there is anything that blocks this PR from being merged into the master.

Asking because there is a good chance that Comms_experience will be working on the ticket that will require the correct modal size to be used in the modal.

Thank you for the help! : )

**Comment by MichaelParadis** [comment]
@igor-m-jobber we are going to address the modal size thing but I will merge this in right now

**Comment by igor-m-jobber** [comment]
@MichaelParadis yeah. I saw your conversation and I think it's a great idea.
Thank you for merging this PR!

**Pull request merged**
